rllib/utils/types.py

```diff
@@ -8,6 +8,11 @@
 # the default trainer config to be used.
 PartialTrainerConfigDict = dict
 
+# Represents a fully filled out config of a Policy class.
+# This is usually the same as TrainerConfigDict, but parts of it may be altered
+# in e.g. a multi-agent setup, where we have >1 Policies in the same Trainer.
+PolicyConfigDict = dict
```

Comment:
I think it would be clearer to just use the same type, otherwise it's quite confusing unless you read the fine print.

Reply:
You can comment in the trainer config that it may be customized per policy

Reply:
Hmm, don't agree actually. If we distinguish between TrainerConfig, PartialTrainerConfig, EnvConfig (which are all just dicts, but with a different functionality within the RLlib setup process), we should have PolicyConfigDict as well.

Reply:
The main difference is that 99% of the time, TrainerConfigDict is literally the same as PolicyConfigDict... perhaps we could call it something like `policy_config: TrainerConfigDict`, I think that makes it fairly clear? The name of the variable also gives information about the semantics. I don't think the type needs to be overly rigorous, in this case being too precise is more likely to harm readability than help.

Reply:
Fair enough. Good idea to re-name the variable itself. In the beginning, I found it often confusing what was what.